eng/Publishing.props

```diff
@@ -1,5 +0,0 @@
-<Project>
```

Comment:
I suppose this wasn't being used anywhere?

Reply:
Correct. PublishingVersion=3 is already the default.